src/test/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/activities/ActivityResourceIntTest.kt

```diff
@@ -203,7 +203,7 @@ class ActivityResourceIntTest : IntegrationTestBase() {
 
     @Test
     fun `will return error if anything fails`() = runTest {
-      activitiesApi.stubGetScheduleWithError(ACTIVITY_SCHEDULE_ID)
+      mappingServer.stubGetMappingsWithError(ACTIVITY_SCHEDULE_ID, 503)
```

Comment:
it was expecting the wrong API call first so actually triggered a `404` instead, which we now map to not found